README.md

```diff
@@ -51,6 +51,9 @@ Once all the tools are installed, there is no need to invoke them directly, ever
 $ make codegen
 ```
 
+However, there are some deviations from the other SDKs, in which a manual code change is needed:
+- user agent parsing does not match for the Mac OS X operating system
```

Comment:
The  manual code changes to the openapi model is for geo, not OS.  Instead of describing it, just say geo.py needs to be manually updated and add a link to this commit.